src/org/labkey/test/components/CustomizeView.java

```diff
@@ -494,19 +494,22 @@ private void removeItem(String fieldKey, ViewItemType type)
         for (WebElement el : elements)
         {
             _driver.fireEvent(el, WebDriverWrapper.SeleniumEvent.blur);
-            for (int i = 0; i < 10; i++)
+            try
             {
-                try
-                {
-                    ScrollUtils.scrollIntoView(el);
-                    builder.moveToElement(el).click().build().perform();
-                }
-                catch (StaleElementReferenceException ignore) {}
-                catch (WebDriverException ignore)
+                for (int i = 0; i < 10; i++)
                 {
-                    ScrollUtils.scrollUnderFloatingHeader(el);
+                    try
+                    {
+                        ScrollUtils.scrollIntoView(el);
+                        builder.moveToElement(el).click().build().perform();
+                    }
+                    catch (WebDriverException ignore)
+                    {
+                        ScrollUtils.scrollUnderFloatingHeader(el);
+                    }
```

Comment:
Could this be one try / catch? That would require logic in the catch block to only call `scrollUnderFloatingHeader` if it was a `WebDriverException`.
Not saying it is a better solution, just asking a question.